src/test/java/seedu/anilist/logic/commands/CommandsListTest.java

```diff
@@ -40,9 +40,9 @@ public void getListOfCommands() {
      */
     private String listOfCommandsToString(String[] listOfCommands) {
         String result = "";
-        for (int i = 0; i < listOfCommands.length; i++) {
+        for (String listOfCommand : listOfCommands) {
```

Comment:
uhh shouldn't it be
command instead of listOfCommand

Reply:
yeap